gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -34,14 +34,6 @@
 """
 
 
-skip_kwargs_validation_for = [
-                                'firewall',
```

Comment:
Is this not needed? we are using this for avoiding validation for some cases right?